README.md

```diff
@@ -167,3 +169,41 @@ server.route({
 ```curl
  curl http://localhost:3000/foo/math/add?a=2&b=2
 ```
+
+prefix the hapi decorations (chairo and hemera co-exist)
```

Comment:
Using chairo and hemera at the same is very special and I dont want to encourage them. Its good that a prefix exist but chairo or seneca should not be mentioned.